ReturnBookUI.java

```diff
@@ -3,40 +3,41 @@
 
 public class ReturnBookUI {
 
-	public static enum UI_STATE { INITIALISED, READY, INSPECTING, COMPLETED };
+	public static enum ReturnBookUiState { INITIALISED, READY, INSPECTING, COMPLETED };
 
-	private ReturnBookControl CoNtRoL;
-	private Scanner input;
-	private UI_STATE StATe;
+	private ReturnBookControl returnBookUiControl;
+	private Scanner keybordInput;

```

Comment:
can you change keybordInput into keyboardInput

Reply:
yep

Reply:
cool